pygotham/talks/models.py

```diff
@@ -43,6 +46,56 @@ def __str__(self):
         return self.name
 
 
+class Speaker(db.Model):
+
+    """Speaker, represents the intent to give a talk"""
+
+    __tablename__ = 'speakers'
+
+    id = db.Column(db.Integer, primary_key=True)
+
+    talk_id = db.Column(
+        db.Integer, db.ForeignKey('talks.id'), nullable=False,)
+    talk = db.relationship(
+        'Talk', backref=db.backref('speakers', lazy='dynamic'),)
+
+    user_id = db.Column(
+        db.Integer, db.ForeignKey('users.id'), nullable=False)
+    user = db.relationship(
+        'User', backref=db.backref('speakers', lazy='dynamic'))
+
+    recording_release = db.Column(db.Boolean, nullable=True)
+    confirmed_ts = db.Column(ArrowType, nullable=True)
```

Comment:
Don't include data types in column names. Just as you wouldn't call a column `id_int` you shouldn't include the `_ts` prefix. You should also look for prior art when naming things. `confirmed_at` is used elsewhere. You'd then use `declined_at` for the other field.